ARM-Quick-Start/README.md

```diff
@@ -1,17 +1,17 @@
 
-# Quickstart: Deploy Azure Video Analyzer for Media with ARM Template 
+# Quickstart: Deploy Azure Video Indexer with ARM Template 
 
 ## Overview
 
-In this Quick-Start you will create an Azure Video Analyzer for Media (formerly Video Indexer) account by using ARM template (PREVIEW)
+In this Quick-Start you will create an Azure Video Indexer (formerly Video Indexer) account by using ARM template (PREVIEW)
 
-The resource will be deployed to your subscription and will create the Azure Video Analyzer for Media resource based on parameters defined in the avam.template file.
+The resource will be deployed to your subscription and will create the Azure Video Indexer resource based on parameters defined in the avam.template file.

```

Comment:
Maybe do a find "avam" and replace with "vi"

Reply:
that's a bit risky as there are code / class names / file refrences with that name :
![image](https://user-images.githubusercontent.com/512709/166677453-57dc257f-f335-4a48-a44c-09c1a4796fda.png)

I think I will avoid it at this time